### DISCUSSION TITLE
Can I use batchrunner to execute an experimental design?

### DISCUSSION THREAD

**Discussion by ghost** [mesa/mesa#1734] [Q&A]
I have more than 10 model parameters and I want to run specific combinations of these parameters for my experiment (Design of Experiment) because it would be time-consuming to run a full factorial experiment. Can I achieve this with batchrunner?

**Comment by tpike3** [comment]
Yes you can. It may be a little tricky depending on how you specify the parameters to minimize the combinatorics possibilities, but you should be able to specify it.

**Reply by ghost** [reply]
Thank you!

**Reply by ghost** [reply]
Is there an example of the usage of batchrunner for DoE among the files on github that you are aware of so that I can refer to it?
